plugins/obs-filters/data/locale/en-US.ini

```diff
@@ -75,7 +75,9 @@ NoiseSuppress.Intensity="Suppression Intensity"
 NoiseSuppress.Method="Method"
 NoiseSuppress.Method.Speex="Speex (low CPU usage, low quality)"
 NoiseSuppress.Method.RNNoise="RNNoise (good quality, more CPU usage)"
-NoiseSuppress.Method.nvafx="NVIDIA Noise Removal (good quality, no CPU usage)"
+NoiseSuppress.Method.Nvafx="NVIDIA Noise Removal FX"
```

Comment:
Should use `NoiseSuppress.Method.Nvafx.Denoise` for the string name for specifying "Noise Removal" now that we have more than one NVAFX method.

Reply:
done

Reply:
done